doc/optimizers/SNOPT.rst

```diff
@@ -13,13 +13,27 @@ BFGS quasi-Newton update.
 
 Installation
 ------------
+
+Building from source
+********************
 SNOPT is available for purchase `here 
 <http://www.sbsi-sol-optimize.com/asp/sol_snopt.htm>`_. Upon purchase, you should receive a zip file. Within the zip file, there is a folder called ``src``. To use SNOPT with pyoptsparse, paste all files from ``src`` except snopth.f into ``pyoptsparse/pySNOPT/source``.
 
 From v2.0 onwards, only SNOPT v7.7.x is officially supported.
 To use pyOptSparse with previous versions of SNOPT, please checkout release v1.2.
 We currently test v7.7.7 and v7.7.1.
 
+Installation by conda
+*********************
+When installing via conda, all pyoptsparse binaries are pre-compiled and installed as part of the package. However, the `snopt` binding module cannot be included as part of the package due to license restrictions.
+
+If you are installing via conda and would like to use SNOPT, you will need to build the `snopt` binding module on your own and inform `pyoptsparse` that it should use that library.
+
+Suppose you have built the binding file, producing `snopt.cpython-310.so`, living in the folder `~/snopt-bind`.
```

Comment:
We generally use `` (double backticks) to display paths and variables etc., as single will make it italic in RST. I suggest updating where appropriate.